openjdk.test.jck/src/test.jck/net/adoptopenjdk/stf/Jck.java

```diff
@@ -0,0 +1,968 @@
+/*******************************************************************************
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+package net.adoptopenjdk.stf; 
+
+import static net.adoptopenjdk.stf.extensions.core.StfCoreExtension.Echo.ECHO_ON; 
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileReader;
+import java.io.InputStream;
+import java.net.InetAddress;
+import java.net.ServerSocket;
+import java.util.Properties;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import net.adoptopenjdk.stf.environment.DirectoryRef; 
+import net.adoptopenjdk.stf.environment.FileRef;
+import net.adoptopenjdk.stf.environment.JavaVersion;
+import net.adoptopenjdk.stf.environment.StfTestArguments; 
+import net.adoptopenjdk.stf.extensions.core.StfCoreExtension;
+import net.adoptopenjdk.stf.plugin.interfaces.StfPluginInterface;
+import net.adoptopenjdk.stf.processes.ExpectedOutcome; 
+import net.adoptopenjdk.stf.processes.StfProcess; 
+import net.adoptopenjdk.stf.runner.modes.HelpTextGenerator; 
+
+public class Jck implements StfPluginInterface{
+	private static final Logger logger = LogManager.getLogger(JavaVersion.class.getName());
+	
+	// Variables which hold the input values
+	private String testJdk;
+	private String tests;
+	private String testExecutionType;
+	private String withAgent;
+	private String extraJvmOptions;
+	private String jckVersion;
+	private String config;
+	private String jckRoot;
+	
+	private enum TestSuite{
+		RUNTIME, COMPILER, DEVTOOLS; 
+	}
+	private TestSuite testSuite;
+	
```

Comment:
Extra line-break should be removed.

Reply:
Need space before the opening brace, in Line 54.

Reply:
Done